aries-site/src/examples/templates/wizard/components/CancellationLayer.js

```diff
@@ -25,7 +28,7 @@ export const CancellationLayer = ({ onSetOpen }) => {
         <Box
           as="footer"
           gap="small"
-          direction="row"
+          direction={size !== 'small' ? 'row' : 'column'}
           align="center"
           justify="end"
```

Comment:
Based on recent guidance, on mobile the button in this box should take full-width -- can we add `fill="horizontal"` when size is small?

Reply:
yes I actually had to add it to `button` but looks better nice catch!